### PR TITLE
Inelegant solution for custom logging FILE pointer

### PR DESCRIPTION
**Pull request by AidanWelch**
This just simply creates a getter and setter function for a `FILE*` where logs are written.  This allows software incorporating libpostal to more easily respond to errors.  There's probably a better way to do this, but I wanted to make minimal changes.

### CODE REVIEW

File: src/vector_math.h [hunk 1]
```diff
@@ -8,10 +8,15 @@
 
 #define ks_lt_index(a, b) ((a).value < (b).value)
 
-#if   defined(USE_SSE)
-#include <emmintrin.h>
+#if defined(USE_SSE)
+	#if defined(USE_NEON)
+		#include "sse2neon.h"
```
Comment (missinglink) [inline src/vector_math.h:13]: Is this change related to the PR?

Comment (AidanWelch) [reply src/vector_math.h:13]: No sorry, that was just another change for a project I'm working, I can create a new branch for this PR and re-open it

Comment (AidanWelch) [reply src/vector_math.h:13]: Re-opened here https://github.com/openvenues/libpostal/pull/669
